MAUI/DataForm/layout.md

```diff
@@ -0,0 +1,372 @@
+---
+layout: post
+title: Layout in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about the Layout support in Syncfusion .NET MAUI DataForm (SfDataForm) control and more.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Layout in .NET MAUI DataForm (SfDataForm)
+
+## Overview
+
+The data form supports linear and grid layouts and manages layout of label, editor, and validation label.
+
+## Linear data form layout
+
+By default, the data form arranges the fields one-by-one. It is applicable for both label positions: left and top.
+
+When the label position is Left, the linear layout is shown as follows:
+
+When the label position is Top, the linear layout is shown as follows:
+
+## Grid data form layout
+
+By default, the data form arranges one data field per row. It is possible to have more than one data fields per row by setting the [ColumnCount] property which provides grid like layout for the data form. The column order of data form field in a row can also be changed using [ItemsOrderInRow].
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+             xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+
+    <dataForm:SfDataForm
+        x:Name="dataForm"
+        ColumnCount="2"
+        >
+    </dataForm:SfDataForm>
+
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.ColumnCount = 2;
+
+{% endhighlight %}
+{% endtabs %}
+
+N> Setting the `ColumnCount` property to SfDataForm does not arrange the data field in a group according to the column count. To set the column count for data fields in the data form group, refer to [loading different layout for data form group]
+
+When the label position is Left, the grid layout is shown as follows:
+
+When the label position is Top, the grid layout is shown as follows:
+
+## Label visibility
+
+The label can be hided by defining the [DataFormDisplayOptions] attribute or by handling `GenerateDataFormItem` event. In this case, only the editor will be loaded.
+
+#### Using attributes
+
+{% tabs %}
+{% highlight C# %}
+private double? percentage;
+
+    [DataFormDisplayOptions(ShowLabel = false)]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Using event
+
+{% tabs %}
+{% highlight C# %}
+
+ this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+        {
+            e.DataFormItem.PlaceholderText = "Enter your name";
+            e.DataFormItem.ShowLabel = false;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Label position
+
+Label can be positioned either top or left side of the editor. By using the [DataFormDefaultLayoutSettings.LabelPosition] property, you can layout the label associated with editor for data form.
```

Comment:
The label can be positioned either top or left side of the editor. By using the [DataFormDefaultLayoutSettings.LabelPosition] property, lay out the label associated with the editor for the data form.

Reply:
Content changed